docs/ops/sequence/OneHot_1.md

```diff
@@ -32,14 +32,14 @@ Types of input scalars `on_value` and `off_value` should match and can be any of
 
 **Inputs**:
 
-* **1**: `indices`: input tensor of rank `N` with indices of any supported integer data type. Can be 0D. Required.
-* **2**: `depth`: scalar (0D tensor) of any supported integer type that specifies number of classes and the size of one-hot dimension.
-* **3**: `on_value`: scalar (0D tensor) of any type that is the value that the locations in output tensor represented by indices in input take.
-* **4**: `off_value`: scalar (0D tensor) of any type that is the value that the locations not represented by indices in input take.
+* **1**: `indices`: input tensor with non-negative indices of any supported integer data type. Can be 0D. Required.
+* **2**: `depth`: positive scalar (0D tensor) of any supported integer type that specifies the number of classes and thus the size of the one-hot dimension. Required.
+* **3**: `on_value`: scalar (0D tensor) of any supported type that fills the locations in output tensor specified in `indices`. Required.
+* **4**: `off_value`: scalar (0D tensor) of the same type as `on_value` that fills the locations not represented in `indices`. Required.
 
 **Outputs**:
 
-* **1** Output tensor of rank `N+1`, where `N` is a rank of input tensor `indices`. A new axis of the size `depth` is inserted at the dimension `axis`.
+* **1** A tensor of rank `N+1`, where `N` is a rank of the input tensor `indices`. A new axis of the size `depth` is inserted at the dimension `axis`. The output type is the same as the `on_value` type.
 
```

Comment:
Add section with supported data types and use them in the description of input/output tensors

Reply:
The types section is added.